dramatiq/actor.py

```diff
@@ -54,15 +54,16 @@ class Actor(Generic[P, R]):
       options(dict): Arbitrary options that are passed to the broker
         and middleware.
     """
+
     def __init__(
         self,
-        fn: Callable[P, Union[R, Awaitable[R]]],
+        fn: Callable[P, R | Awaitable[R]],
```

Comment:
be aware this things will make older python version to be incompatible too...

Reply:
@bonastreyair what do you think?